beer-garden/functionary
# Add scheduler logging and vscode debug task

This adds a vscode debug task and enables logging for the scheduler.

## Test Instructions
* Start everything as usual.
* View the logs on the scheduler container.  You should see messages from beat about what it's doing.
* In vscode there's a new "run_scheduler" debug task.  You can launch that to start the scheduler locally in debug mode.
